package.json

```diff
@@ -23,6 +23,7 @@
     "lint": "yarn prettier --list-different",
     "lint-fix": "yarn prettier --write",
     "prettier": "prettier .",
+    "clean": "rm -rf node_modules cache cache-zk artifacts artifacts-zk dist typechain",
```

Comment:
```suggestion
    "clean": "for dir in node_modules cache cache-zk artifacts artifacts-zk dist typechain; do mv \"${dir}\" \"_${dir}\"; rm -rf \"_${dir}\" &; done",
```

Reply:
^^ fwiw the above simply moves the dirs to somewhere that we don't care about (prefixes with _), and then starts a background rm -rf on the newly moved directory. This is faster in the case where there are a lot of files to be nuked.

Reply:
I'm no expert on bash so i'm curious why your changes are better? I noticed you used something similar in other repos but I didn't understand why